ansible/roles/web/tasks/frontend-js.yml

```diff
@@ -0,0 +1,27 @@
+- name: Add yarn repository key
+  apt_key:
+    url: https://dl.yarnpkg.com/debian/pubkey.gpg
+    state: present
+
+- name: Add yarn repository
+  apt_repository:
+    repo: deb https://dl.yarnpkg.com/debian/ stable main
+    state: present
+
+- name: Add NodeSource repository key
+  apt_key:
+    url: https://deb.nodesource.com/gpgkey/nodesource.gpg.key
+    state: present
+
+- name: Add node.js repository from NodeSource
+  apt_repository:
+    repo: deb https://deb.nodesource.com/node_12.x xenial main
```

Comment:
I would stick to NodeJS 10 till NodeJS 12 is promoted as LTS. Node 12 was just released a few days ago and it still may have regressions with the dependencies with WebPack.

Module authors usually take a while to ensure compatibility with a new release.